src/tables.rs

```diff
@@ -192,6 +192,211 @@ impl BlockSize {
     }
 }
 
+pub const fn dav1d_txfm_size<const txsize_discr: usize>() -> TxfmSize {
+    let Some(size) = TxfmSize::from_repr(txsize_discr) else {
```

Comment:
Any reason not to use `.unwrap_unchecked()` here?

Reply:
Ahhh it looks like it is still unstable. Gotcha.

Reply:
> Ahhh it looks like it is still unstable. Gotcha.

Finally stable in 1.83!

Reply:
Were you maybe waiting for this to merge the PR?